### PR TITLE
Document Current Websocket Support

### PR DESCRIPTION
**Pull request by h0ngcha0**
image

### CODE REVIEW

File: docs/sdk/block.md [hunk 1]
```diff
@@ -162,4 +162,73 @@ subscription.subscribe()
...
+```
+
+Here is an example of how to interact with it using the
+[ws](https://www.npmjs.com/package/ws) package in Typescript:
+
+
+```typescript
+import WebSocket from 'ws';
+
+const ws = new WebSocket('wss://node.mainnet.alephium.org/events');
```
Comment (polarker) [inline docs/sdk/block.md]: We should not expose this. It has quite some overhead. Also, I am not sure our rate limit could monitor WS properly.

File: docs/sdk/block.md [hunk 2]
```diff
@@ -162,4 +162,73 @@ subscription.subscribe()
 
 // unsubscribe
 subscription.unsubscribe()
+```
+
+### WebSocket Support
+
+Alephium full node also supports streaming blocks using
+WebSocket. Here is an example using
+[wscat](https://github.com/websockets/wscat) from terminal:
```
Comment (polarker) [inline docs/sdk/block.md:176]: Let's add a remark that this is under active development, and breaking changes might be expected.

File: docs/sdk/block.md (hunk 1 above)
Comment (h0ngcha0) [reply docs/sdk/block.md]: Will close.

Comment (h0ngcha0) [reply docs/sdk/block.md]: Closed now

**Pull request merged**